### ISSUE TITLE
EM fisherySector update

### ISSUE THREAD

**Issue by neilriley-NOAA** [nwfsc-fram/boatnet#2062]
@sethgerou-noaa 

I am trying to square these fishery and fisherySector lookups against the structure we discussed for the OTS lookups. Not sure this is going to transfer exactly. We can change the fisherySector description from "Whiting" to ' Shoreside Hake".  

![image](https://user-images.githubusercontent.com/31421964/100925778-52df3d00-3497-11eb-9221-3939df6b72fb.png)

**Comment by sethgerou-noaa** [comment]
OK, this change has been made.

**Comment by neilriley-NOAA** [comment]
been trying to figure out how to map the OTS fishery to the EM fishery. I think we can map the fisheries below possibly using the an alias table.  

LE trawl   ITQ EM | Electronic Monitoring
-- | --
LE trawl ITQ observer | Catch Shares


Then if we add MSCV to the fisherySector, SSH and MSCV could reside in both fishery and fisherySector. I am not sure if that would work or not?

**Comment by sethgerou-noaa** [comment]
not sure I'm getting this - I think we may want to discuss when you have time.

**Comment by neilriley-NOAA** [comment]
@sethgerou-noaa 

I was chatting with Scott and he agreed we should code this with the most flexibility as possible for OTS observer selection. It is unlikely there would be an observer the same time as a camera. But for the LE sablefish fishery, we should code to selection assuming there will be different selection rates for each fisherySector.

**Comment by sethgerou-noaa** [comment]
I believe we've resolved this

**Comment by neilriley-NOAA** [comment]
@sethgerou-noaa Yes we have. Should I provide comments on the mockups in this ticket or create another one?

**Comment by sethgerou-noaa** [comment]
Y

**Comment by sethgerou-noaa** [comment]
@neilriley-NOAA this is good place for mockup comments

**Comment by neilriley-NOAA** [comment]
@sethgerou-noaa I like how this would works it's way through the options. If it was implemented live, would it have the same look?  Curious what @scottleach-NOAA and @johnlafargue-noaa think of it?

**Comment by sethgerou-noaa** [comment]
@neilriley-NOAA Yes, I built the mockup within the app, so the UI can look exactly as it does right now, and we can fine tune as needed.

**Comment by johnlafargue-noaa** [comment]
I like the way it looks.  I'll be interested to see what fixed gear fishermen say about "LE Trawl ITQ". I still think "LE ITQ" might be better, but we should see what they say.

**Comment by neilriley-NOAA** [comment]
@scottleach-NOAA @johnlafargue-noaa @sethgerou-noaa 

I am going to put something on the calendar so we can discuss final comments/concerns before Seth codes this into the app.

**Comment by neilriley-NOAA** [comment]
@sethgerou-noaa I wanted to check in see if there are any decisions you are waiting on from us for this ticket?

**Comment by sethgerou-noaa** [comment]
@neilriley-NOAA I don't believe I need anything else.  Hoping to work on this later in the week.

**Comment by neilriley-NOAA** [comment]
Sounds good. Thanks!
